src/App.tsx

```diff
@@ -1,17 +1,14 @@
 import React from "react";
-import NameForm from "./components/BalanceForm";
 import Chart from "./components/Chart";
 import StatedBalanceForm from "./components/StatedBalanceForm";
-import {getValues, saveValues} from "./core/Memory";
+import BalanceService from "./components/BalanceService";
 
 
 const App = () => (
     <div className="App">
-        <StatedBalanceForm getValues={getValues} saveValues={saveValues}/>
-
+        <StatedBalanceForm balanceService={BalanceService.instance}/>
```

Comment:
@BartNeoteric what do you think about using singletons in code like this?
Do you think we should create factories to create our components?